### DISCUSSION TITLE
remove merged points of a plane

### DISCUSSION THREAD

**Discussion by ghazalbangash** [pyvista/pyvista#3298] [Q&A] [answered]
Hi, 
I have been having a little problem working with merge and boolean differences.Basically I merge a plane into a mesh using the merge function and then, later on, I want to remove these points from the mesh but I don't know the range to give it to the remove_points function since the plane is not triangular I can't use boolean_difference. Can someone guide me? I want to do this cause I want to use vtk.vtkSelectPolyData() and it gives me "can't follow edge" error when I merge a plane into my mesh. Thank you for your time

**Comment by banesullivan** [comment, marked as answer]
Perhaps the `clean` filter is what you need?: https://docs.pyvista.org/api/core/_autosummary/pyvista.PolyData.clean.html#pyvista.PolyData.clean

**Reply by ghazalbangash** [reply]
Thank you a lot, fixed my problem.
